.github/workflows/scripts/chart-integration/lib.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+LOG_PATH=.
+
+function init_codegen() {
+    # init var
+    CHART_MOUNT=/home/$USER_ID/charts-mnt/codegen
+    CHART_MOUNT_STR=$(echo $CHART_MOUNT | sed 's/\//\\\//g') # add a '\' before every '/' in CHART_MOUNT
+    MODELREPO=m-a-p
+    MODELNAME=OpenCodeInterpreter-DS-6.7B
+    MODELID=$MODELREPO/$MODELNAME
+    MODELDOWNLOADID=models--$MODELREPO--$MODELNAME
+
+    ### PREPARE MODEL
+    # check if the model is already downloaded
+    if [ -d "$CHART_MOUNT/$MODELDOWNLOADID" ]; then
+        echo "Model $MODELID already downloaded!"
+        USE_MODELDOWNLOADID=True
+    else
+        echo "Downloading model $MODELID..."
+        MODELDIR=$CHART_MOUNT/$MODELNAME
+        if [ ! -d "$MODELDIR" ]; then
+            mkdir -p $MODELDIR
+        fi
+        huggingface-cli download $MODELID --local-dir $MODELDIR --local-dir-use-symlinks False
+        USE_MODELDOWNLOADID=False
+    fi
+
+    ### CONFIG VALUES.YAML
+    # set image name to the CI images, replace opea/gen-ai-comps with amr-registry.caas.intel.com/aiops/opea-ci
+    sed -i "s/opea\/gen-ai-comps/amr-registry.caas.intel.com\/aiops\/opea-ci/g" values.yaml
```

Comment:
we can use sed -i "s#pattern#replace#g" to increase code readability

Reply:
thanks. changed according to this comment.